client/dashboard/sites/logs/components/style.scss

```diff
@@ -1,19 +1,9 @@
 .site-logs-details-modal {
-	&__row {
-		display: grid;
-		grid-template-columns: 180px 1fr;
-		column-gap: 16px;
-		align-items: start;
-	}
-
-	&__field-title {
-		white-space: nowrap;
-		font-weight: 500;
-		font-size: 0.75rem;
-		text-transform: uppercase;
-	}
 
-	&__field-value {
-		word-break: break-word;
+	.dataforms-layouts-panel__field-control button {
```

Comment:
Could we use custom render function instead of overriding the styles?

Reply:
A custom render function for each field, similar to what is done for the 'Request Type' for example? This adds HTML, but as a child of the Button component, so the default button styling still applies. Even with custom HTML we'd still need to try to avoid the the lack of text wrapping in particular:
<img width="522" height="273" alt="Screenshot 2025-10-03 at 08 13 06" src="https://github.com/user-attachments/assets/b8a23b0f-57c5-403b-a4f2-f2cfe6a02928" />

Reply:
I see you're AFK at the moment, so I've deployed -- I'm happy to follow-up if needed though.